packages/orbit-components/src/primitives/TooltipPrimitive/TooltipPrimitive.ct-story.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import cx from "clsx";
+
+import { PLACEMENTS } from "../../common/placements";
+import Stack from "../../Stack";
+import Text from "../../Text";
+import { Icons } from "../..";
+import type { Props, Size } from "./types";
+
+import TooltipPrimitive from ".";
+
+export const tooltipContentCtStory = (
+  <Stack>
+    <Text>
+      We take security very seriously. Especially when it comes to your personal and sensitive
+      details.
+    </Text>
+  </Stack>
+);
+
+type TooltipPrimitiveProps = Pick<Props, "placement" | "block" | "error" | "help">;
+
+export function DefaultTooltipPrimitive({
+  placement = PLACEMENTS.BOTTOM,
+  error,
+  help,
+  block,
+}: TooltipPrimitiveProps) {
+  return (
+    <div
+      className={cx([
+        "flex min-h-[128px]",
```

Comment:
We can add some padding here, so the tooltip can "breathe" better. We probably need a bigger min-height as well.

Reply:
there was set bigger height before, I wanted to avoid "big blank spaces", but side-padding will be good (especially for small mobile viewport).